azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilterAutoConfiguration.java

```diff
@@ -23,16 +23,16 @@
 @Configuration
 @ConditionalOnWebApplication
 @ConditionalOnProperty(prefix = "azure.activedirectory", value = {"client-id", "client-secret"})
-@EnableConfigurationProperties({AADAuthenticationFilterProperties.class, ServiceEndpointsProperties.class})
+@EnableConfigurationProperties({AADAuthenticationProperties.class, ServiceEndpointsProperties.class})
 public class AADAuthenticationFilterAutoConfiguration {
-    private static final Logger LOG = LoggerFactory.getLogger(AADAuthenticationFilterProperties.class);
+    private static final Logger LOG = LoggerFactory.getLogger(AADAuthenticationProperties.class);
 
-    private final AADAuthenticationFilterProperties aadAuthFilterProperties;
+    private final AADAuthenticationProperties aadAuthFilterProperties;
     private final ServiceEndpointsProperties serviceEndpointsProperties;
```

Comment:
serviceEndpointsProp(s)?

Reply:
Have updated all the variables names etc. to keep consistency, thanks.